torch/utils/_sympy/interp.py

```diff
@@ -178,3 +153,33 @@ def sympy_interp(
     except Exception:
         log.warning("failed while executing %s(%s)", handler_name, args)
         raise
+
```

Comment:
refactor so it's easier to hash cons without having this file depend on dict/nodes